src/pytest_ansible/module_dispatcher/__init__.py

```diff
@@ -1,6 +1,6 @@
 """Define BaseModuleDispatcher class."""
 
-from collections.abc import Sequence
+from typing import Sequence  # noqa: UP035
```

Comment:
If we're doing this why not change `ruff`'s `target-version` to `"py38"`?

Reply:
perfect, didn't think about that :)